source/includes/fundamentals/code-snippets/CRUD/bulkOps.go

```diff
@@ -87,13 +90,15 @@ func main() {
 
 	fmt.Println("\nReplaceOneModel:\n")
 	{
+		// Creates a write model that specifies a replace operation
 		// begin bulk replace model
 		models := []mongo.WriteModel{
 			mongo.NewReplaceOneModel().SetFilter(bson.D{{"title", "Lucy"}}).
 				SetReplacement(Book{Title: "On Beauty", Author: "Zadie Smith", Length: 473}),
 		}
 		// end bulk replace model
 
+		// Performs the bulk write and prints the number of replaced documents
```

Comment:
S: same as above; applies elsewhere
```suggestion
		// Runs the bulk write operation and prints the number of replaced documents
```